packages/components/src/components/form/Checkbox/Checkbox.tsx

```diff
@@ -171,7 +176,10 @@ export const Checkbox = ({
     const theme = useTheme();
 
     const handleKeyUp = (event: KeyboardEvent<HTMLElement>) => {
-        if (event.code === KEYBOARD_CODE.SPACE || event.code === KEYBOARD_CODE.ENTER) {
+        if (
+            isDisabled === false &&
```

Comment:
nit: `isDisabled === false` not needed as there is default value and it cannot be `undefined` => `!isDisabled` is ok

Reply:
True! Thanks!